pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -85,8 +85,8 @@ var (
 	OperatorVersionAMQOnline           OperatorVersion = "1.4"
 	OperatorVersionMonitoring          OperatorVersion = "1.1.5"
 	OperatorVersionSolutionExplorer    OperatorVersion = "0.0.55"
-	OperatorVersionRHSSO               OperatorVersion = "10.0.0"
-	OperatorVersionRHSSOUser           OperatorVersion = "10.0.0"
+	OperatorVersionRHSSO               OperatorVersion = "9.0.2"
```

Comment:
should these be `9.0.3`?

Reply:
Good question. `9.0.3` is our own release of the keycloak operator from our fork (https://github.com/integr8ly/keycloak-operator-rhmi). There is no 9.0.3 release from the upstream keycloak-operator (https://github.com/keycloak/keycloak-operator/releases). Either one is fine in my opinion and I leaned towards `9.0.2` initially as this maps to the upstream version.